client/packages/system/src/Item/Components/StockItemSearchInputWithStats/StockItemSearchInputWithStats.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import {
+  useToggle,
+  useFormatNumber,
+  useTranslation,
+  Autocomplete,
+  defaultOptionMapper,
+  Tooltip,
+} from '@openmsupply-client/common';
+import { useStockItemsWithStats } from '../../api';
+import {
+  itemFilterOptions,
+  StockItemSearchInputWithStatsProps,
+} from '../../utils';
+import { getItemOptionRenderer } from '../ItemOptionRenderer';
+
+export const StockItemSearchInputWithStats = ({
```

Comment:
This was removed with page, have added back... can probably be refactored sometime..